analyzers/tests/SonarAnalyzer.Test/TestCases/DeadStores.RoslynCfg.cs

```diff
@@ -192,7 +192,7 @@ void calculateRate2(int a, int b)
                 x = 11; // FN, muted due to try/catch
                 x = 12;
                 Console.Write(x);
-                x = 13; // Compliant, Console.Write could throw and value is read fater try/catch
+                x = 13; // FN, muted due to try/catch
```

Comment:
I'd add a more explanative comment here (and later) - mentioning the change in LVA.

Reply:
Not needed -you are fixing this later.